### PR TITLE
Implemented Full Adder and Subtractor from Universal gates

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/Interactive-Book/issues/35
Links to : 
1. Full adder - https://circuitverse.org/users/13891/projects/45277
2. Full subtractor - https://circuitverse.org/users/13891/projects/45278

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@ankitkataria Have a look

**Comment by JoshHeng** [comment]
Preview

File: docs/Combinational/full_adder.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ The full adder is a three input and two output combinational circuit.
 
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/247" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
+## Full adder from Universal Gates
```
Comment (JoshHeng) [inline docs/Combinational/full_adder.md]: Maybe change to `Full Adder From Universal Gates` (note capitals)

File: docs/Combinational/full_sub.md [hunk 2]
```diff
@@ -23,3 +23,7 @@ A is the 'minuend', B is 'subtrahend', C is the 'borrow' produced by the previou
 <div style="text-align:center"><img src="../../assets/images/fullsubstrator_circuitdiagram.jpg" /></div>
 
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/12119" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
+
+## Full Subtractor from Universal Gates
```
Comment (JoshHeng) [inline docs/Combinational/full_sub.md]: Maybe change to `Full Subtractor From Universal Gates` (note capitals)

**Comment by aayush-05** [comment]
Good to be merged @tachyons

**Pull request merged**